R/google_linters.R

```diff
@@ -0,0 +1,9 @@
+#' @title Linters for the Google extension to the tidyverse style guide
```

Comment:
Can we drop the @title tag? See the examples here:
https://style.tidyverse.org/documentation.html